addons/packages/antrea/1.5.2/bundle/config/schema.yaml

```diff
@@ -0,0 +1,123 @@
+#! schema.yaml
+
+#@data/values-schema
+#@schema/desc "OpenAPIv3 Schema for antrea"
+---
+#@schema/desc "The cloud provider in use. One of the following options => aws, azure, vsphere, docker"
+infraProvider: vsphere
+#@schema/desc "NodeSelector configuration applied to all the deployments"
+#@schema/type any=True
+nodeSelector:
+deployment:
+  #@schema/desc "Update strategy of deployments"
+  #@schema/nullable
+  updateStrategy: ""
+  rollingUpdate:
+    #@schema/desc "The maxUnavailable of rollingUpdate. Applied only if RollingUpdate is used as updateStrategy"
+    #@schema/nullable
+    maxUnavailable: 1
+    #@schema/desc "The maxSurge of rollingUpdate. Applied only if RollingUpdate is used as updateStrategy"
+    #@schema/nullable
+    maxSurge: 0
+daemonset:
+  #@schema/desc "Update strategy of daemonsets"
+  #@schema/nullable
+  updateStrategy: ""
+antrea:
+  #@schema/desc "Configuration for antrea"
+  config:
+    #@schema/desc "Control SNAT IPs of Pod egress traffic."
+    egress:
+      #@schema/desc "The CIDR ranges to which outbound Pod traffic will not be SNAT'd by Egresses."
+      exceptCIDRs:
+      - ""
+    nodePortLocal:
+      #@schema/desc "Enable NodePortLocal feature."
+      enabled: false
+      #@schema/desc "Provide the port range used by NodePortLocal."
+      portRange: ""
+    #@schema/desc "AntreaProxy related configuration options."
+    antreaProxy:
+      #@schema/desc "ProxyAll tells antrea-agent to proxy all Service traffic."
+      proxyAll: false
+      #@schema/desc "Specifies the host IPv4/IPv6 addresses for NodePort."
+      nodePortAddresses:
+       - ""
+      #@schema/desc "List of Services which should be ignored by AntreaProxy."
+      skipServices:
+        - ""
+      #@schema/desc "Load-balance traffic destined to the External IPs of LoadBalancer services."
+      proxyLoadBalancerIPs: false
+    #@schema/desc "Provide the address of Kubernetes apiserver."
+    #@schema/nullable
+    kubeAPIServerOverride: ""
+    #@schema/desc "The name of the interface on Node which is used for tunneling or routing the traffic."
+    #@schema/nullable
+    transportInterface: ""
+    #@schema/desc "The network CIDRs of the interface on Node which is used for tunneling or routing the traffic."
+    transportInterfaceCIDRs:
+      - ""
+    #@schema/desc "The names of the interfaces on Nodes that are used to forward multicast traffic."
+    multicastInterfaces:
+      - ""
+    #@schema/desc "Tunnel protocols used for encapsulating traffic across Nodes. One of the following options => geneve, vxlan, gre, stt"
+    tunnelType: none
+    #@schema/desc "Determines how tunnel traffic is encrypted. One of the following options => none, ipsec, wireGuard"
```

Comment:
Sorry for the late comments at the stage. I think we removed the ipsec support from Antrea commercial release image. This is because of the US export regulation. If TCE is picking up the open-source Antrea image, we can keep the ipsec in the description.